ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -100,6 +100,7 @@ private[apiserver] final class ApiUserManagementService(
       } { case (user, pRights) =>
         for {
           _ <- identityProviderExistsOrError(user.identityProviderId)
+          _ <- userBelongsToIDPOrError(user.identityProviderId, user.id)
```

Comment:
This probably doesn't make sense for the create user rpc

Reply:
its this funny case which we discussed already, if there is a user exists with a different IDP assigned - it should throw a permission denied